<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>91</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>$1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>71</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>$zero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>$at</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>$v0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>$v1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>$a0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>$a1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>$a2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>$a3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>$t0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>$t1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>$t2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>$t3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>$t4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>$t5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>$t6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>$t7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>$s0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>$s1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>$s2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_52">
       <property name="text">
        <string>$s3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_53">
       <property name="text">
        <string>$s4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_55">
       <property name="text">
        <string>$s5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_56">
       <property name="text">
        <string>$s6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_57">
       <property name="text">
        <string>$s7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_58">
       <property name="text">
        <string>$t8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_59">
       <property name="text">
        <string>$t9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_62">
       <property name="text">
        <string>$k0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_63">
       <property name="text">
        <string>$k1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_61">
       <property name="text">
        <string>$gp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_60">
       <property name="text">
        <string>$sp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_54">
       <property name="text">
        <string>$fp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_64">
       <property name="text">
        <string>$ra</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Btn_ni">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Instruction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>611</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>pc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pc">
    <property name="geometry">
     <rect>
      <x>516</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_help">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>help</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_c">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>countinue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_w">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>SUBCMD</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>520</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTEMU">
    <property name="title">
     <string>TEMU</string>
    </property>
   </widget>
   <addaction name="menuTEMU"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
